news/2020-19-11-mappers-report-september-october.md

```diff
@@ -0,0 +1,211 @@
+
+His recent ranked map, being his second, [Suzuyu - Euphorium](https://osu.ppy.sh/beatmapsets/1254692#fruits/2607602) features a simple yet engrossing finesse through the apt variation of flow and patterning, as well as the perfect coherence attention to the music. Not only does he convey his fixation to eroge, [M2U feat. Guriri - Marigold (VIP Remix)](https://osu.ppy.sh/beatmapsets/1257779#fruits/2614247) evinces that -Joakh can project technical mapping fashion as well! Showcasing peculiar stream shapes and diverse flow usage, it is a must-see for fruit catchers!
+
+As we all know, eroge mappers are destined for greatness! With more time and experience, -Joakh can definitely be one of the top mappers in osu!catch! Something that is worth mentioning would also be his capability of mapping high bpm songs such as his take on [UNDEAD CORPORATION - The Empress scream off ver](https://osu.ppy.sh/beatmapsets/1285068#fruits/2668183). We hope to see more of -Joakh's maps and hopefully his motivation further grows as he is unquestionably a very promising mapper!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+Did someone ask for contests? No? Ahh well, you get them anyways! As always we inform you about any interesting mapping related contests going on. Even if you do not plan to participate in them, you might want to check out if any of them has an open voting phase going on! Don't forget: Every vote counts!
+
+And don't forget. Should you have any interesting contest planned or know about, let us know! We would love to give them more attention in our news post!
+
+### PDC2020
+
+Good news! The classic **Pending Cup** mapping contest is back for another round and currently accepting submissions! Time is tight though, as the current deadline for submission is **November 8th**. If you were thinking of entering but forgot to submit, take this as a reminder to do so - and if this is your first time hearing about it, maybe try it out and see how it goes! The Pending Cup has always been a source of awesome maps each year, and we expect this year will be no different. So even if you don't want to enter, be sure to stay tuned to see who comes out on top! You can find detailed information about PDC 2020 in its [dedicated forum thread](https://osu.ppy.sh/community/forums/topics/1155108).
+
+### Monthly Beatmapping Contest
+
+The Monthly Beatmapping Contest series is continuing as strongly as ever, albeit with some changse, but also with some awesome results to share! Starting with MBC #8, the contest has changed to a judge-based format. Where before screeners would pick maps for the community to vote on, now screeners function the same but the final vote comes down to a panel of judges with detailed impressions and scoring. For MBC #9 the formula was further mixed up by making the contest have a **theme** instead of a **restriction**. With these changes we get to see an ever evolving regular contest format to stay excited about!
```

Comment:
```suggestion
The Monthly Beatmapping Contest series is continuing as strongly as ever, albeit with some changes, but also with some awesome results to share! Starting with MBC \#8, the contest has changed to a judge-based format. Where before screeners would pick maps for the community to vote on, now screeners function the same but the final vote comes down to a panel of judges with detailed impressions and scoring. For MBC \#9 the formula was further mixed up by making the contest have a **theme** instead of a **restriction**. With these changes, we get to see an ever-evolving regular contest format to stay excited about!
```
fix typo & escaping
add a hyphen to properly form a compound adjective